extensions/tokenizers/src/test/java/ai/djl/huggingface/tokenizers/HuggingFaceTokenizerTest.java

```diff
@@ -389,4 +389,29 @@ public void testTruncationAndPaddingForPairInputs() throws IOException {
             Assert.assertEquals(encoding.getIds().length, 8);
         }
     }
+
+    @Test
+    public void testSpecialTokenHandling() throws IOException {
```

Comment:
We might not need this test, the test above this already has special characters.